app/src/main/cpp/skyline/services/audio/IAudioDevice.cpp

```diff
@@ -26,26 +26,32 @@ namespace skyline::service::audio {
             state.process->WriteMemory(deviceName.c_str(), request.outputBuf.at(0).address + offset, deviceName.size() + 1);
             offset += deviceName.size() + 1;
         }
+        return {};
     }
 
-    void IAudioDevice::SetAudioDeviceOutputVolume(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response) {}
+    Result IAudioDevice::SetAudioDeviceOutputVolume(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response) {
+        return {};
+    }
 
-    void IAudioDevice::GetActiveAudioDeviceName(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response) {
+    Result IAudioDevice::GetActiveAudioDeviceName(type::KSession &session, ipc::IpcRequest &request, ipc::IpcResponse &response) {
         std::string deviceName("AudioStereoJackOutput");
 
         if (deviceName.size() > request.outputBuf.at(0).size)
             throw exception("Too small a buffer supplied to GetActiveAudioDeviceName");
```

Comment:
Is there not an appropriate result for this?

Reply:
Doesn't seem to be, ryu just returns success